### DISCUSSION TITLE
記事へのリンク一覧の折り畳みメッセージについて

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#133] [要望・アイデア・不具合]
記事の下部にある「この記事は以下の記事からリンクされています」のリンク一覧についてです。

５件以上のリンクがあると、「過去のN件を表示する」という折り畳みを開くメッセージが表示されます。

![closed](https://user-images.githubusercontent.com/15214393/152789920-7ef75ea9-473d-4f7e-b14d-6fbbb3743eb0.png)

この折り畳みを開くと、一番下に「最新のN件以外を隠す」というメッセージが表示されます。

![opend](https://user-images.githubusercontent.com/15214393/152789935-f6d3ac10-165d-4753-9130-9dc88d33a05b.png)

これは「最新の５件以外を隠す」（5は固定値）あるいは「過去のN件を隠す」が正しいのではないでしょうか？

（英語版でも "Hide links except the latest N ones" と、同じようなメッセージになっています）

また、リンクがちょうど5件の場合には、「過去の0件を表示する」と表示されますが、これは不要ではないでしょうか？

![zero](https://user-images.githubusercontent.com/15214393/152790287-79cf499d-4498-43a5-b652-2ae83ea1bf57.png)

細かい話ですが、ご確認いただければと思います。

**Comment by kyntk** [comment]
@yoshi389111 ありがとうございます!
こちら調査と対応いたします!

**Comment by ohakutsu** [comment]
@yoshi389111 
おまたせしました！こちら修正が完了しました！
リリースノートはこちらになります 💁‍♂️ 
https://qiita.com/release-notes#2022-03-02
